CONTRIBUTING.md

```diff
@@ -0,0 +1,136 @@
+## Contributing to Tasties project
+
+---
+
+Hello and welcome to Tasties project. 
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved.
+
+
+
+### Quicklinks
+
+---
+
+- [Pre-Requirements](#pre-requirements)
+- [Issues](#issues)
+- [Pull Requests](#pull-requests)
+- [Testing](#testing)
+- [Code of Conduct](#code-of-conduct)
+- [Getting Help](#getting-help)
+
+
+### Pre Requirements
+
+---
+
+To contribute to our project please download and install the following:
+
+- Vagrant.
+- VirtualBox(or other VM that works with "Vagrant").
+
+After the installations type 'vagrant up' and automatically the development environment will set up.
```

Comment:
This is vague, from which location does the vagrant up should run? (It is a basic thing that everyone should know, but if you weren't familiar with Vagrant, what would you do?)